src/Standards/Squiz/Tests/Commenting/FunctionCommentUnitTest.inc

```diff
@@ -1006,6 +1006,11 @@ function foo($foo) {}
  */
 public function foo($a, $b) {}
 
+/**
```

Comment:
Why did you place this test in the middle of the `inheritDoc` test instead of at the end of the file ?